src/libostree/ostree-repo-pull-private.h

```diff
@@ -74,7 +74,8 @@ typedef struct {
   GBytes           *summary_data;
   GBytes           *summary_data_sig;
   GVariant         *summary;
-  GHashTable       *summary_deltas_checksums;
+  GHashTable       *summary_deltas_checksums; /* Filled from summary and delta indexes */
```

Comment:
IIUC, this should be

```suggestion
  GHashTable       *summary_deltas_checksums; /* Filled from summary or delta indexes */
```

 right?

Reply:
Well, the code does handle the case where they come from both. I mean, we don't really *expect* that to happen, but it could (if mixing version in some weird way maybe), and we handle it.